orm/src/test/java/org/hibernate/search/test/configuration/IndexManagerOverrideTest.java

```diff
@@ -27,14 +27,18 @@
  */
 public class IndexManagerOverrideTest {
 
+	public static class FooIndexManager extends DirectoryBasedIndexManager {
+
+	}
+
 	@Test
 	public void verifyIndexExclusivity() {
 		FullTextSessionBuilder builder = new FullTextSessionBuilder();
 		FullTextSession ftSession = builder
 			.setProperty( "hibernate.search.Book.indexmanager", "near-real-time" )
 			.setProperty(
 					"hibernate.search." + Foo.class.getName() + ".indexmanager",
-					"org.hibernate.search.testsupport.indexmanager.RamIndexManager"
+					FooIndexManager.class.getName()
```

Comment:
Ok, I was just wondering. My reason to use the Ram IndexManager often, is to try to keep the test execution quick.
(The default one creates stuff on disk, and it gets slow when thousand of tests do it)